rust/tvm-sys/src/datatype.rs

```diff
@@ -73,6 +73,18 @@ impl DataType {
     pub fn lanes(&self) -> usize {
         self.lanes as usize
     }
+
+    pub const fn int(bits: u8, lanes: u16) -> DataType {
+        DataType::new(DL_INT_CODE, bits, lanes)
+    }
+
+    pub const fn float(bits: u8, lanes: u16) -> DataType {
+        DataType::new(DL_FLOAT_CODE, bits, lanes)
+    }
+
+    pub const fn uint(bits: u8, lanes: u16) -> DataType {
+        DataType::new(DL_FLOAT_CODE, bits, lanes)
```

Comment:
Should this be `DL_FLOAT_CODE`?
```suggestion
        DataType::new(DL_UINT_CODE, bits, lanes)
```